demisto_sdk/commands/common/tools.py

```diff
@@ -830,6 +830,29 @@ def get_file(
         return {}
 
 
+def get_file_or_remote(file_path: Path, clear_cache=False):
+    content_path = get_content_path()
+    relative_file_path = None
+    if file_path.is_absolute():
+        absolute_file_path = file_path
+        try:
+            relative_file_path = file_path.relative_to(content_path)
+        except ValueError:
+            logger.warning(f"Could not get relative path for {file_path}")
```

Comment:
what do we do on this case? Let's log it so someone reading the logs can understand the implication